src/lib/y2partitioner/dialogs/partition_size.rb

```diff
@@ -1,7 +1,43 @@
+require "y2storage"
 require "yast"
 require "cwm/dialog"
 require "ui/greasemonkey"
 
+Yast.import "Popup"
+
+module Y2Storage
+  # Monkey-patched Region to get a size
+  class Region
+    def size
+      block_size * length
+    end
+
+    def last
```

Comment:
https://github.com/yast/yast-storage-ng/blob/master/src/lib/y2storage/region.rb#L61
is not enough?